packages/talos_certifier/src/ports/message.rs

```diff
@@ -16,7 +16,8 @@ pub trait MessageReciever: SharedPortTraits {
     async fn subscribe(&self) -> Result<(), SystemServiceError>;
     async fn commit(&self) -> Result<(), SystemServiceError>;
     fn commit_async(&self) -> Option<JoinHandle<Result<(), SystemServiceError>>>;
-    async fn update_savepoint(&mut self, offset: i64) -> Result<(), SystemServiceError>;
+    fn update_savepoint(&mut self, offset: i64) -> Result<(), Box<SystemServiceError>>;
+    async fn update_savepoint_async(&mut self, offset: i64) -> Result<(), SystemServiceError>;
```

Comment:
did you mean safepoint? "f" letter vs "v" ?

Reply:
When I wrote the function initially, I was thinking of something like a DB Savepoint. But that is not correct. I will rename that in the next PR. Thanks for pointing this out.